tools/bin/make/airbyte_ci_dev_install.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Check if pyenv is installed, and install it if not
+if ! which pyenv >/dev/null; then
+    echo "pyenv not found, installing pyenv..."
+    brew install pyenv
```

Comment:
I'd rather you fail with an error message telling the user to `brew install pyenv`. Perhaps that's just me.

Reply:
I agree